packages/lit-node-client-nodejs/src/index.ts

```diff
@@ -1,14 +1,11 @@
-import { isNode, log } from '@lit-protocol/misc';
+import 'cross-fetch/polyfill';
```

Comment:
do we think its worth it to check after the polyfill loading for `fetch.polyfill` to be defined? 
from looking at their implementations they do set this `boolean` to true if its found a polyfill is needed.
https://github.com/lquixada/cross-fetch/blob/v4.x/src/node-polyfill.js

Reply:
What should we do with that information if we check for it?  I'm not sure it's relevant for us, though I guess a consumer might want to check it for some reason?

Reply:
if its false for some reason then i think we might want to warn that the polyfill might have failed. so we could do our own check on the global for fetch and conditionally import, and then check if `polyfill` is set to true. The purpose is to just warn end consumers that fetch might have failed polyfill.